.template/addons/crud/lib/templates/slim/scaffold/_form.html.slim.tt

```diff
@@ -0,0 +1,28 @@
+= form_with(model: <%= model_resource_name %>) do |form|
+  - if <%= singular_table_name %>.errors.any?
+    div style="color: red"
+      h2 = "#{pluralize(<%= singular_table_name %>.errors.count, "error")} prohibited this <%= singular_table_name %> from being saved:"
+      ul
+        - <%= singular_table_name %>.errors.each do |error|
+          li = error.full_message
+
+<% attributes.each do |attribute| -%>
+  .mb-3
+<% if attribute.password_digest? -%>
+    = form.label :password, class: 'form-label'
+    = form.password_field :password, class: 'form-control'
+
+  .mt-3
```

Comment:
In this case, I think it's odd to have the `mt-3` right after the `mb-3` element, what do you think? 🤔

Reply:
Nice catch 👍 It should be `.mb-3` instead. Fixed in https://github.com/nimblehq/rails-templates/pull/506/commits/7f2273c716eae67c317b227f89e738b32fa8d016